### DISCUSSION TITLE
Supportability Enhancements

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#235] [Ideas]
I've had an issue lately where, if I leave the container running for more than 24 hours, it just craps out. I've looked at the following things to check on the health and compare before/after readings:

- docker stats
- docker system df
- docker top viseron
- nvidia-smi -q

None really point to the issue. 

[There is a small app running on the same box but it is a no-GPU-resource app. When viseron craps out, the other app continues to function. ]

I'm also monitoring, in Home Assistant, the `sensor.viseron*` and, when it stops detecting, the sensor believes everything is working. (When I "stop" the container, I do get an alert as well as when I restart the container.)  But the easy to tell it has stopped working is it doesn't detect the stationary items. (So I'll do a parameter walk to confirm it isn't detecting.) So I have the habit of just restarting it twice a day.

And on the issue of restarting. Sometimes when it is restarted, one camera's detection is sent to all the other cameras. (Something mentioned previously as a bug.) 

In those mentioned situations, is there something I can monitor to verify functionality? Is it possible to have a MQTT channel that might offer stats that could be monitored? 

Not a priority as I have my work-arounds (restarting). Just a suggestion for the future.

**Thank you again!**

**Comment by roflcoopter** [comment]
I am sorry you are having issues.
I am away for the weekend and need to finish some other things up, then i will see what i can do about this

**Reply by gniknalu** [reply]
The issues aren't severe - I have work-arounds. Hope you enjoy your weekend. 

These were **just suggestions** for future enhancements, not complaints.

Thanks again!
